Vagrantfile

```diff
@@ -69,12 +69,16 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
   end
 
   # Kubernetes master
-  config.vm.define "master" do |config|
-    customize_vm config
-
-    config.vm.provision "shell", run: "always", path: "#{ENV['KUBE_TEMP']}/master-start.sh"
-    config.vm.network "private_network", ip: "#{$master_ip}"
-    config.vm.hostname = ENV['MASTER_NAME']
+  config.vm.define "master" do |c|
+    customize_vm c
+    if ENV['KUBE_TEMP'] then
+      script = "#{ENV['KUBE_TEMP']}/master-start.sh"
+      c.vm.provision "shell", run: "always", path: script
+    else
+      puts "\e[31mWARNING: Kube provisioner not set\e[0m""]]"
```

Comment:
Remove the puts to the log.

We enforce that USING_KUBE_SCRIPTS is required above when doing an up, so users should not run into an issue of doing a vagrant up where the var is not set.  Outputing messages to the console for normal interaction makes it seem like something is broken.